simple/stats/mcf_importer.py

```diff
@@ -51,20 +52,29 @@ def do_import(self) -> None:
       if self.is_main_dc:
         self.output_fh.write_string(self.input_fh.read_string())
       else:
-        triples = list(
-            map(lambda x: _to_triple(x),
-                mcf_to_triples(self.input_fh.read_string_io())))
+        parser_triples: list[list[str]] = []
+        # DCID references
+        dcids: dict[str, str] = {}
+        for parser_triple in mcf_to_triples(self.input_fh.read_string_io()):
+          [subject_id, predicate, value, _] = parser_triple
+          if predicate == _DCID:
+            dcids[subject_id] = value
+          parser_triples.append(parser_triple)
+
+        triples = list(map(lambda x: _to_triple(x, dcids), parser_triples))
```

Comment:
maybe use a helper func? (given 3 levels of indent)

Reply:
Done.